frontend/src/features/imprint_page/routes/ImprintPage.tsx

```diff
@@ -0,0 +1,41 @@
+import Footer from '@/components/Layout/Footer';
+import PageLayout from '@/components/Layout/PageLayout';
+import ContactUs from '@/features/landing_page/components/ContactUs';
+import { useTranslation } from 'react-i18next';
+
+export function ImprintPage() {
+  const { t } = useTranslation(['imprint']);
+
+  return (
+    <PageLayout styleNames="body-font relative w-full text-center">
+      <h1 className="title-font mb-8 text-3xl font-medium text-primary-500 dark:text-primary-300 sm:text-4xl">
+        {t('imprint:title')}
+      </h1>
+      <section
+        id="general"
+        className="mx-auto mb-4 flex flex-col text-base leading-relaxed lg:w-2/3"
+      >
+        <p>PermaplanT</p>
+        <p>{t('imprint:chairman')}: Markus Raab</p>
+        <p>ZVR Nr.: {t('imprint:zvr_placeholder')}</p>
+        <p>
+          E-Mail: <a href="mailto://contact@permaplant.net">contact@permaplant.net</a>
```

Comment:
ZVR Nr., E-Mail, Address needs to be translated, too.

Reply:
I don't think that we need to translate ZVR and E-Mail since the first is an austria specific noun and E-Mail is used in both german and english. But I will add Address to the words to be translated.

Reply:
Yeah, I agree it is already quite pedantic. I pushed the change. Please check if I did it correctly.

@absurd-turtle can you review and merge? The contact form is not yet translated, will you do it in another PR?